gitops/overlays/prod/configs/frontend/config.conf

```diff
@@ -8,7 +8,7 @@ OTEL_TRACES_ENDPOINT=https://dynatrace.prod-dp.admin.dts-stn.com/e/676a0299-9802
 #
 # Application feature flags configuration
 #
-ENABLED_FEATURES=hcaptcha,power-platform-status-checker
+ENABLED_FEATURES=hcaptcha
```

Comment:
This will affect `PROD` if we remove it.

Reply:
Re-added `power-platform-status-checker` to config